src/patternfly/components/Form/examples/Form.md

```diff
@@ -87,6 +87,16 @@ cssPrefix: pf-c-form
       This is helper text for an invalid input
     {{/form-helper-text}}
   {{/form-group}}
+  {{#> form-group}}
+    {{#> form-label form-label--attribute=(concat 'for="' form--id '-simple-form-comment"')}}
+      Comment
+    {{/form-label}}
+    {{#> form-control controlType="input" input="true" form-control--value="This is a valid comment" form-control--modifier="pf-m-validated" form-control--attribute=(concat 'type="text" id="' form--id '-simple-form-comment" name="' form--id '-simple-form-comment" aria-describedby="' form--id '-simple-form-comment-help"')}}
+    {{/form-control}}
+    {{#> form-helper-text form-helper-text--modifier="pf-m-validated" form-helper-text--attribute=(concat 'id="' form--id '-simple-form-comment-help" aria-live="polite"')}}
+      This is helper text for validated input
+    {{/form-helper-text}}
+  {{/form-group}}
```

Comment:
Can you also add an example(s) to the form control component examples? Is this intended to work with selects and textareas, too?

Reply:
Yes, and done.